--- configure.ac	2012-01-05 13:36:57.000000000 +0100
+++ configure.ac.oden	2012-01-25 16:52:41.004902944 +0100
@@ -310,7 +310,7 @@
 	MYSQL_inc_check="$withval/include/mysql"],
 	    [MYSQL_lib_check="/usr/local/mysql/lib/mysql /usr/local/lib/mysql /opt/mysql/lib/mysql \
 		/usr/lib/mysql /usr/lib64/mysql /usr/local/mysql/lib /usr/local/lib /opt/mysql/lib /usr/lib \
-		/usr/sfw/lib/"
+		/usr/sfw/lib/ /usr/lib64"
 	MYSQL_inc_check="/usr/local/mysql/include/mysql /usr/local/include/mysql \
 		/opt/mysql/include/mysql /opt/mysql/include /usr/include/mysql /usr/sfw/include/mysql"])
 		AC_ARG_WITH(mysql-lib,
@@ -335,6 +335,7 @@
 	fi
 	case "$MYSQL_libdir" in
                  /usr/lib ) MYSQL_lib="" ;;
+                 /usr/lib64 ) MYSQL_lib="" ;;
 	  /* ) MYSQL_lib=-L$MYSQL_libdir; LDFLAGS="$MYSQL_lib $LDFLAGS";;
 	  * )  AC_MSG_ERROR([The MySQL library directory ($MYSQL_libdir) must be an absolute path.]) ;;
 	esac
@@ -504,7 +505,7 @@
             [  --with-unixodbc=<path>     root directory path of unixodbc installation],
             [UNIXODBC_lib_check="$withval/lib/unixodbc $with_unixodbc/lib $withval/lib"
              UNIXODBC_inc_check="$withval/include/unixodbc"],
-            [UNIXODBC_lib_check="/usr/local/unixodbc/lib/unixodbc /usr/local/lib/unixodbc /usr/lib/unixodbc /usr/local/unixodbc/lib /usr/local/lib /opt/unixodbc/lib /usr/lib"
+            [UNIXODBC_lib_check="/usr/local/unixodbc/lib/unixodbc /usr/local/lib/unixodbc /usr/lib/unixodbc /usr/local/unixodbc/lib /usr/local/lib /opt/unixodbc/lib /usr/lib /usr/lib64"
              UNIXODBC_inc_check="/usr/local/unixodbc/include/unixodbc /usr/local/include/unixodbc/ /usr/local/include /opt/unixodbc/include/unixodbc /opt/unixodbc/include /usr/include/ /usr/include/unixodbc"])
         AC_ARG_WITH(unixodbc-lib,
             [  --with-unixodbc-lib=<path> directory path of unixodbc library installation],
@@ -527,6 +528,7 @@
         fi
         case "$UNIXODBC_libdir" in
            /usr/lib ) UNIXODBC_lib="" ;;
+           /usr/lib64 ) UNIXODBC_lib="" ;;
           /* ) UNIXODBC_lib="-L$UNIXODBC_libdir -Wl,-rpath,$UNIXODBC_libdir"
                LDFLAGS="$UNIXODBC_lib $LDFLAGS"
                ;;
