--- modules/luabackend/Makefile.am	2012-01-05 13:36:05.000000000 +0100
+++ modules/luabackend/Makefile.am.oden	2012-01-25 17:34:10.264778032 +0100
@@ -2,10 +2,9 @@
 EXTRA_DIST=OBJECTFILES OBJECTLIBS 
 
 INCLUDES=-I/usr/include/lua5.1
-#INCLUDES=-I/usr/local/include/luajit-2.0 -DUSE_LUAJIT
 
 lib_LTLIBRARIES = libluabackend.la
 
 libluabackend_la_SOURCES=luabackend.cc luabackend.hh minimal.cc reload.cc lua_functions.cc master.cc private.cc slave.cc supermaster.cc dnssec.cc 
-libluabackend_la_LDFLAGS=-module -avoid-version -llua5.1
-#-lluajit-5.1
+libluabackend_la_LDFLAGS=-module -avoid-version
+libldapbackend_la_LIBADD =-llua5.1
